CONTRIBUTING.md

```diff
@@ -70,8 +77,8 @@ Now you can make your changes locally.
 6.  When you're done making changes, check that your changes pass all tests
 
 ```bash
-pytest -x .
-flake8 .
+poe test
+poe check
```

Comment:
```suggestion
poetry run poe test
poetry run poe check
```

... just in case the guy forgot to activate poetry shell.

Reply:
not necessary, poe does it for you if you (assuming it's installed in the host environment).